js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetCheckAutomationList.tsx

```diff
@@ -0,0 +1,121 @@
+import {Box, CursorHistoryControls} from '@dagster-io/ui-components';
+import {useMemo} from 'react';
+
+import {FIFTEEN_SECONDS, useQueryRefreshAtInterval} from '../../app/QueryRefresh';
+import {EvaluationList} from '../AutoMaterializePolicyPage/EvaluationList';
+import {AssetKey} from '../types';
+import {
+  AssetCheckAutomationListQuery,
+  AssetCheckAutomationListQueryVariables,
+} from './types/AssetCheckAutomationList.types';
+import {gql} from '../../apollo-client';
+import {useCursorPaginatedQuery} from '../../runs/useCursorPaginatedQuery';
+import {ASSET_CONDITION_EVALUATION_RECORD_FRAGMENT} from '../AutoMaterializePolicyPage/GetEvaluationsQuery';
+import {AssetCheckKeyFragment} from './types/AssetChecksQuery.types';
+
+interface Props {
+  assetCheck: AssetCheckKeyFragment;
+  checkName: string;
+}
+
+export const AssetCheckAutomationList = ({assetCheck, checkName}: Props) => {
+  const {queryResult, evaluations, paginationProps} = useAssetCheckEvaluationsQueryResult({
+    assetKey: assetCheck.assetKey,
+    checkName,
+  });
+
+  useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
+
+  return (
+    <>
+      <Box
+        padding={{vertical: 12, horizontal: 20}}
+        flex={{direction: 'row', justifyContent: 'flex-end'}}
+      >
+        <CursorHistoryControls {...paginationProps} style={{marginTop: 0}} />
+      </Box>
+      <EvaluationList
```

Comment:
It seems like `<EvaluationList />` was already in the codebase, but not in use? Was thinking the `useAssetCheckEvaluationsQueryResult` hook could be used elsewhere but maybe this is the only spot!

Reply:
I built it using Storybook, and I'm adding live callsites in this PR stack.

There is a similar (but different) hook for querying evaluations on assets instead of asset checks.